### PR TITLE
Renaming files

### LINKED ISSUE #31: Example should be renamed to include simpletest
Example `motor_servo_sweep.py` should be renamed to `motor_servo_sweep_simpletest.py` to fit with CircuitPython library standards.

### PR DESCRIPTION
**Pull request by williamlai2**
Closes https://github.com/adafruit/Adafruit_CircuitPython_Motor/issues/31

### CODE REVIEW

File: docs/examples.rst [hunk 1]
```diff
@@ -3,8 +3,8 @@ Simple tests
 
 Ensure your device works with this simple test.
 
-.. literalinclude:: ../examples/motor_servo_sweep.py
-    :caption: examplesmotor_servo_sweep.py
+.. literalinclude:: ../examples/motor_servo_sweep_simpletest.py
+    :caption: examplesmotor_servo_sweep_simpletest.py
```
Comment (kattni) [inline docs/examples.rst]: There was a typo in the original. Please add a `/` to the `:caption:` so it reads 
```
:caption: examples/motor_servo_sweep_simpletest.py
```

**Review by kattni — CHANGES_REQUESTED** [review]
This is great! Everything is passing. There is one thing I'd like changed - there was a typo in the original that carried over to your update. Please see the comment below. Once that's fixed, it'll be all set to merge!

**Comment by williamlai2** [comment]
Thanks. I have fixed that typo now.

**Review by kattni — APPROVED** [review]
Thank you so much for sticking with this! Great job!

**Pull request merged**